frontend/packages/dev-console/src/components/topology/TopologyControlBar.tsx

```diff
@@ -15,8 +16,8 @@ interface TopologyControlBarProps {
   visualization: Visualization;
 }
 
-const TopologyControlBar: React.FC<TopologyControlBarProps> = ({ visualization }) => {
-  const layout = visualization.getGraph()?.getLayout() ?? 'COLA_LAYOUT';
+const TopologyControlBar: React.FC<TopologyControlBarProps> = observer(({ visualization }) => {
```

Comment:
`observer` makes the component a `memo`. You can remove your call to memo below.

Reply:
Fixed. Thanks.

Reply:
Hmm interesting.. reading the docs.. :)